docs/configuration.md

```diff
@@ -42,8 +42,11 @@ The following table lists the available properties along with their default valu
 Unless you need to set a non-default value, it is recommended to only populate overridden properties in the custom `.env`.
 
 | Name                                  | Default       | Description                                                                                                                                                                                                                                                                                            |
-|---------------------------------------| ------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ------------------------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `CLIENT_TRANSPORT_SECURITY`           | "false"       | Flag to enable or disable TLS for both networks.                                                                                                                                                                                                                                                       |
+| `CLIENT_TRANSACTION_RESET`            | "50"          | Number of transaction executions, until client reinitialization.                                                                                                                                                                                                                                       |
+| `CLIENT_DURATION_RESET`               | "3600000"     | Time until client reinitialization. (ms)                                                                                                                                                                                                                                                               |
+| `CLIENT_ERROR_RESET`                  | "100"         | Number of error to be reached, until client reinitialization.                                                                                                                                                                                                                                          |
```

Comment:
I am wondering why would we need to tolerate `100` errors before re-init. I would set this to `1`, since once we start getting errors, we should recycle/reset the sdkClient instance, and if we were to set it to `1`. why would we need this property and  logic  around  it on the first place?

Reply:
All those values are just suggestions from me, I should've wrote in the description that I will appreciate any suggestions for more appropriate values. Also yeah 100 might be a lot, but error could also be `UNKNOWN` or `CONTRACT REVERT` or something else. Or should I just limit this check only for `INVALID_TRANSACTION_BODY` ?

Reply:
My suggestion: we might want to keep the `CLIENT_ERROR_RESET` property for all other errors but leave a hardcoded check exclusively for `INVALID_TRANSACTION_BODY` and as soon as we identify `1` occurrence of this, refresh the sdkClient instance.

Why? over the past 2 days our initial workaround has been live, I've noticed that there are some occurrences of `INVALID_TRANSACTION_BODY` and when they come, they come in group, right after another, presumably until the next refresh happens:


<img width="283" alt="image" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/123040664/34487704-4a89-4e95-b4d3-39a92d64d8ac">

Reply:
Okay that make sense. Will do it. Thank you for the suggestion

Reply:
Going with this approach of reset on 1, I'd say instead of a number, the config value should be a list of HAPI response types, if the error is a match then reset. In this way should find another error type that requires a reinitialize we can add it to the list